### PR TITLE
Sync ramsete example with java ramsete-command example

### PR DESCRIPTION
**Pull request by fletch3555**

### CODE REVIEW

File: ramsete/robotcontainer.py [hunk 1]
```diff
@@ -4,66 +4,74 @@
...
+                    -self.driverController.getRightX(),
                 ),
                 self.robotDrive,
             )
         )
 
+    def configureButtons(self):
+        """
+        Use this method to define your button->command mappings. Buttons can be created by
+        instantiating a {@link edu.wpi.first.wpilibj.GenericHID} or one of its subclasses ({@link
```
Comment (virtuald) [inline ramsete/robotcontainer.py:59]: Java links

File: ramsete/robot.py [hunk 2]
```diff
@@ -14,10 +14,10 @@
 
 class MyRobot(commands2.TimedCommandRobot):
     """
-    Our default robot class, pass it to wpilib.run
-
-    Command v2 robots are encouraged to inherit from TimedCommandRobot, which
-    has an implementation of robotPeriodic which runs the scheduler for you
+    The VM is configured to automatically run this class, and to call the functions corresponding to
+    each mode, as described in the TimedRobot documentation. If you change the name of this class or
+    the package after creating this project, you must also update the build.gradle file in the
```
Comment (virtuald) [inline ramsete/robot.py:19]: This comment is java-specific. No VM, no build.gradle. Revert to original comment.

Comment (fletch3555) [reply ramsete/robot.py]: Good catch.

File: ramsete/robotcontainer.py (hunk 1 above)
Comment (fletch3555) [reply ramsete/robotcontainer.py]: Ugh git... I fixed that once but then did some git jiggerypokery and I guess it got reverted.  Fixed!

**Pull request merged**